crates/core/tedge_mapper/src/c8y/sm_mapper.rs

```diff
@@ -115,9 +127,9 @@ where
         let () = self.http_proxy.init().await?;
 
         info!("Running");
-        let () = self.publish_supported_log_types().await?;
-        let () = self.publish_get_pending_operations().await?;
-        let () = self.ask_software_list().await?;
+        let () = self.publish_supported_log_types().await?; // moved
+        let () = self.publish_get_pending_operations().await?; // moved
+        let () = self.ask_software_list().await?; // moved
```

Comment:
I don't understand the status of this file.

* Many methods have been moved to `converter.rs`.
* Some others have not.
* And those ones are tagged moved but are still here.

Reply:
Yes, I intend to move some of them back to separate file and name the file accordingly.